aleph-client/src/multisig.rs

```diff
@@ -44,6 +42,8 @@ pub enum MultisigError {
     CallConflict,
     #[error("👪❌ Only the author can cancel aggregation.")]
     NotAuthor,
+    #[error("👪❌ The connection is signed by an account that doesn't match to any member.")]
+    IncorrectSignature,
```

Comment:
nit: rename since IncorrectSignature means at least to me that something is bad with the signature itself not that it was signed by not related to Multisignature account

Reply:
sure, renamed